src/resources/filters/customnodes/theorem.lua

```diff
@@ -88,6 +88,22 @@ _quarto.ast.add_handler({
   end
 })
 
+local included_typst_theorems = false
+local letted_typst_theorem = {}
+local function ensure_typst_theorems(reftype)
+  if not included_typst_theorems then
+    included_typst_theorems = true
+    quarto.doc.include_text("in-header", "#import \"@preview/ctheorems:1.1.0\": *")
+    quarto.doc.include_text("in-header", "#show: thmrules")
+  end
+  if not letted_typst_theorem[reftype] then
+    letted_typst_theorem[reftype] = true
+    local theorem_type = theorem_types[reftype]
+    quarto.doc.include_text("in-header", "#let " .. theorem_type.env .. " = thmbox(\"" .. theorem_type.env .. "\", \"" .. theorem_type.title .. "\", fill: rgb(\"#eeffee\"))")
```

Comment:
> The green is in the example code I used a few lines down

Would you look at that. Reading! 🤦

Reply:
to be fair it was an extremely long line of code (wrapped now)